src/validation/requestSchema.js

```diff
@@ -11,6 +11,17 @@ const requestTripSchema = Joi.object({
   accommodation: JoiValidator.validateString().required(),
 });
 
+const searchRequestTripSchema = Joi.object({
+  id: JoiValidator.validateString(),
+  type: JoiValidator.validateAlphabet(),
```

Comment:
The `type` schema is not validating `one-way`. Please refactor out the type schema on line 5 so as to reuse the schema in this line `(line 16)` and line 5.

Reply:
Okay, I will fix the one-way validation, please can you share more light on "Please refactor out the type schema on line 5 so as to reuse the schema in this line (line 16) and line 5."